test/WebJobs.Script.Tests.Integration/SamplesEndToEndTests.cs

```diff
@@ -972,6 +972,22 @@ public SamplesEndToEndTests(TestFixture fixture)
             Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
         }
 
+        [Fact]
+        public async Task IgnoresFunctionsWithListenerExceptions()
+        {
+            var logs = await TestHelpers.GetFunctionLogsAsync("ServiceBusListenerException", false);
+            // could be first run, '404 not found' takes a while to register
+            if (logs.Count == 0)
+            {
+                await Task.Delay(10000);
```

Comment:
Use TestHelpers.Await to wait for the logs. See examples elsewhere. That way it'll be less flaky and will run faster